DxDispatch/doc/Guide.md

```diff
@@ -332,24 +334,37 @@ You can initialize a buffer is using a sequence. The example below will write `[
 
 ### Buffer: File Data Initializer
 
-You can initialize a buffer using a raw binary file (.dat/.bin) or NumPy array file (.npy).
+You can initialize a buffer using a raw binary file (.dat/.bin), NumPy array file (.npy), or image file (.png, .jpg).
 
 - The `sourcePath` must exist, either relative to the base .json file or the current directory.
 - The `initialValuesDataType` is irrelevant when reading from .npy's since they contain their data type, but when reading from a raw binary file, the type must be given and not `"UNKNOWN"`.
+- When initializing from an image, `initialValuesDataType` determines the final element data type and can be different from the stored pixel format. For example, a 24bpp BGR image can be converted into a buffer of FLOAT32 elements.
 
+Example of initializing a buffer from a raw numpy array:
 ```json
 {
     "initialValues": { "sourcePath": "inputFile.npy" }
 }
 ```
 
+Example of initializing a buffer from a raw binary file:
 ```json
 {
     "initialValuesDataType": "FLOAT32",
     "initialValues": { "sourcePath": "inputFile.dat" }
 }
 ```
 
+Example of initializing a buffer from a JPEG image (the image will resampled from its original format to 800x600 with 3 channels and FLOAT32 elements):
+```json
+{
+    "initialValues": { "sourcePath": "inputFile.jpg" },
+    "initialValuesDataType": "FLOAT32",
+    "resampleSize": [1, 3, 800, 600],
```

Comment:
I have some models that take `[1, 224, 224, 3]` (which yes, is different from typical ONNX). I'm guessing there is no hidden option for that? (possible future consideration...)

Reply:
Unfortunately not right now. Just keeping it explicitly NCHW for now to avoid complexity.